travis/setup_dependencies_common.sh

```diff
@@ -9,6 +9,10 @@ conda config --add channels defaults
 
 shopt -s nocasematch
 
+LATEST_ASTROPY_STABLE=1.1
+LATEST_ASTROPY_LTS=1.0
+LATEST_NUMPY_STABLE=1.11
```

Comment:
Also it can be a plan to rename these as "minimum stable" (e.g. 1.1 for astropy and 1.10 for numpy), to allow some flexibility and only upgrade with pip when something is messed up terribly.